.github/workflows/stubtest_stdlib.yml

```diff
@@ -1,16 +1,16 @@
-name: Run stubtest on the stdlib
+name: Run stubtest
 
 on:
   workflow_dispatch:
   push:
     branches:
       - main
       - master
-    pull_request:
-      paths:
-        - 'stdlib/**'
-        - '.github/workflows/**'
-        - 'tests/**'
+  pull_request:
+    paths:
+      - 'stdlib/**'
+      - '.github/workflows/stubtest_stdlib.yml'
+      - 'tests/stubtest_stdlib.py'
```

Comment:
I'd rather just keep this as `tests/**`, in case we update the test file to use colors.py or something.

Reply:
Fair point, I wasn't 100% sure about that one.

Reply:
```suggestion
      - 'tests/**'
```